### ISSUE TITLE
A custom item list

### ISSUE THREAD

**Issue by asdf337** [Openarl/PathOfBuilding#689]
Program is awesome, thanks for making it! One thing that is really annoying tho is having to remake all the items for every char, it would be nice if the program saved the items uve already made earlier so we can use them again in the future, the custom rare/magic items that is. Thanks :)

**Comment by qikink** [comment]
This is supported by the box on the right, in the item page called "Shared Items:".
![image](https://user-images.githubusercontent.com/2025166/33726538-2de6d37e-db2a-11e7-855b-29124e00d10a.png)

**Comment by asdf337** [comment]
Yes, my bad I figured out there was one already. Thank u for the amazing work with the program, LOVE IT
